pkg/types/installconfig.go

```diff
@@ -85,6 +92,13 @@ type InstallConfig struct {
 	// +optional
 	AdditionalTrustBundle string `json:"additionalTrustBundle,omitempty"`
 
+	// AdditionalTrustBundlePolicy defines conditional policy to add AdditionalTrustBundle
+	// to the nodes' trusted certificate store. If omitted, "ProxyOnly" is considered.
+	// The field can be set to following specified values.
+	// "ProxyOnly" : uses AdditionalTrustBundle when http/https proxy is configured.
+	// "Always" : ignores all conditions and uses AdditionalTrustBundle.
+	AdditionalTrustBundlePolicy string `json:"additionalTrustBundlePolicy,omitempty"`
```

Comment:
If we are going to set this as an enum, this should be typed similar to:

https://github.com/openshift/installer/blob/d10d64c9843e37f500bcb8d66caf339e940b918f/pkg/types/azure/platform.go#L93-L110